src/ant/include/ant/AntennaChecker.hh

```diff
@@ -94,6 +153,9 @@ class AntennaChecker
 
   void findMaxWireLength();
 
+  vector<Violation> getAntennaViolations2(dbNet* net,
```

Comment:
After finishing the reviews, I believe we will deprecate the old code, so we'll not have two functions with the same name.
@maliberty do you think we should completely remove the old code?